app/client/models/errors.ts

```diff
@@ -163,7 +165,10 @@ export function reportError(err: Error|string, ev?: ErrorEvent): void {
       // This is explicitly a user error, or one in the "Client Error" range, so treat it as user
       // error rather than a bug. Using message as the key causes same-message notifications to
       // replace previous ones rather than accumulate.
-      const options: Partial<INotifyOptions> = {key: (err as UserError).key || message};
+      const options: Partial<INotifyOptions> = {
+        key: (err as UserError).key || message,
+        level: (err as UserError).level
```

Comment:
The comment above makes it sound like this is always a user error. Are some user errors not actually errors? I see you labelled this new error a "warning". Does it make the toast show up yellow instead of red?

Reply:
Yes, user errors are shown like normal popups (all black).

Reply:
Since `UserError` is no longer errors only, would it be too annoying to rename the class to `UserAlerts`?

Reply:
It was always used like that. I don't what to change unrelated code here, in this PR. So I can either:
- proceed with this PR and propose a task to do this refactor
- show this popup differently, by directly calling `reportWarning` method that I introduced some time ago

But for me personally, a warning like this, falls into our UserError case. 

WDYT?